koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -499,87 +297,14 @@ def handler(self, src, target_info, arch, output_template, scratch=False,
         self.osbs().wait_for_build_to_get_scheduled(build_id)
         self.logger.debug("Build was scheduled")
 
-        osbs_logs_dir = self.resultdir()
-        koji.ensuredir(osbs_logs_dir)
-        pid = os.fork()
-        if pid:
-            self._incremental_upload_logs(pid)
-
-        else:
-            full_output_name = os.path.join(osbs_logs_dir,
-                                            'openshift-incremental.log')
-
-            # Make sure curl is initialized again otherwise connections via SSL
-            # fails with NSS error -8023 and curl_multi.info_read()
-            # returns error code 35 (SSL CONNECT failed).
-            # See http://permalink.gmane.org/gmane.comp.web.curl.library/38759
-            self._osbs = None
-            self.logger.debug("Running pycurl global cleanup")
-            pycurl.global_cleanup()
-
-            # Following retry code is here mainly to workaround bug which causes
-            # connection drop while reading logs after about 5 minutes.
-            # OpenShift bug with description:
-            # https://github.com/openshift/origin/issues/2348
-            # and upstream bug in Kubernetes:
-            # https://github.com/GoogleCloudPlatform/kubernetes/issues/9013
-            retry = 0
-            max_retries = 30
-            while retry < max_retries:
-                try:
-                    self._write_incremental_logs(build_id,
-                                                 full_output_name)
-                except Exception, error:
-                    self.logger.info("Error while saving incremental logs "
-                                     "(retry #%d): %s", retry, error)
-                    retry += 1
-                    time.sleep(10)
-                    continue
-                break
-            else:
-                self.logger.info("Gave up trying to save incremental logs "
-                                 "after #%d retries.", retry)
-                os._exit(1)
-            os._exit(0)
-
```

Comment:
The whole chunk above, which uses _incremental_upload_logs() and _write_incremental_logs(), needs to stay. This is what makes watching build logs work.